editoast/authz/src/regulator.rs

```diff
@@ -94,6 +94,11 @@ pub trait StorageDriver: Clone {
         >,
     > + Send;
 
+    fn delete_user(
+        &self,
+        identity: &UserIdentity,
```

Comment:
```suggestion
        user_id: i64,
```

This is a DB interface, we should identify rows using PKs by default (unless another behavior is explicitly needed like in `get_user_info_by_identity` for instance).

Reply:
Done